backend/.env.sample

```diff
@@ -1,4 +1,11 @@
-JWT_SECRET=secret 
-MONGO_URL=mongodb://127.0.0.1:27017/talk-trove 
-PORT=5000
-NODE_ENV=dev
\ No newline at end of file
+# JWT_SECRET=secret 
+# MONGO_URL=mongodb://127.0.0.1:27017/talk-trove 
+# PORT=5000
+# NODE_ENV=dev
+
+JWT_SECRET="Anything"
+MONGO_URL="mongodb://127.0.0.1:27017/talk-trove" #default
+PORT=5000 #default
+NODE_ENV=""
+EMAIL_USERNAME="" #sender email
```

Comment:
what is this for Email username and password

Reply:
These fields are sender email credentials. For sending the email containing OTP to reset password, this email address is used as sender.
For EMAIL_USERNAME just give email address And for EMAIL_PASSWORD, you have to create application specific password from your Google account. I have given the steps below.